src/Services/Identity/Identity.API/Services/ProfileService.cs

```diff
@@ -21,6 +21,8 @@ public ProfileService(UserManager<ApplicationUser> userManager)
             _userManager = userManager;
         }
 
+        //New Method ArgumentNullException.ThrowIfNull (obj) Added to .NET 6
```

Comment:
I'm sure this source code is not a guide for C# new features, it is more about architecture.
Why do we need to include links to posts? Why not official documentation then?